spec/models/event_spec.rb

```diff
@@ -16,9 +16,14 @@
   context '.not_happened' do
     let!(:happened)     { create(:event_happened) }
     let!(:not_happened) { create(:event_not_happened) }
+    let!(:today)        { create(:event_today) }
 
     it 'only one event' do
-      expect(Event.not_happened.count).to eq 1
+      expect(Event.not_happened.count).to eq 2
+    end
+
+    it 'include today events' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.